src/ui/units/collections/components/CollectionActions/CollectionActions.tsx

```diff
@@ -135,29 +135,29 @@ export const CollectionActions = React.memo<Props>(
 
         return (
             <div className={b(null, className)}>
-                {collectionData && (
-                    <React.Fragment>
-                        {collectionsAccessEnabled && collectionData.permissions.listAccessBindings && (
+                {collectionData && collectionData.permissions.move && (
+                    <Tooltip content={i18n('action_move')}>
+                        <Button className={b('move')} onClick={onMoveClick}>
```

Comment:
I don’t know why, but after closing the dialog tooltips appear again:
https://github.com/datalens-tech/datalens-ui/assets/29424746/61931f75-33ba-4af0-9669-204b02a856d4

The same problem does not exist on the workbook page. Need to wrap `Buttons` to `div`, and problem will be resolved:

```tsx
<Tooltip content={i18n('action_move')}>
    <div className={b('move')}>
        <Button onClick={onMoveClick}>
            <Icon data={ArrowRight} />
        </Button>
    </div>
</Tooltip>

...

<Tooltip content={i18n('action_access')}>
    <div className={b('access')}>
        <Button onClick={onEditAccessClick}>
            <Icon data={LockOpen} />
        </Button>
    </div>
</Tooltip>
```

And add `display: flex;` to:
```scss
.dl-collection-actions {
    display: flex;
    
    ...
    
}
```

Reply:
fixed